code/game/turfs/open.dm

```diff
@@ -208,6 +208,9 @@
 	if(wet_time > MAXIMUM_WET_TIME)
 		wet_time = MAXIMUM_WET_TIME
 	if(wet == TURF_WET_ICE && air.temperature > T0C)
+		for(var/obj/O in contents)
+			if(O.is_frozen)
+				O.make_unfrozen()
```

Comment:
https://github.com/tgstation/tgstation/pull/22054/files#diff-2939b84f8f6de56f3b4a70ef33122366R138See the